src/XliffTasks/Model/ResxDocument.cs

```diff
@@ -72,5 +74,46 @@ public override void RewriteRelativePathsToAbsolute(string sourceFullPath)
                 }
             }
         }
+
+        /// <summary>
+        /// Attempts to match formatting placeholders as documented at
+        /// https://docs.microsoft.com/en-us/dotnet/standard/base-types/composite-formatting.
+        /// Explanation:
+        /// \{
+        ///    A placeholder starts with an open curly brace. Since curly braces are used in
+        ///    regex syntax we escape it to be clear that we mean a literal {.
+        ///
+        /// (\d+)
+        ///    The "index" component; one or more decimal digits. This is captured in a group
+        ///    to facilitate extracting the numeric value.
+        ///
+        /// (\,\-?\d+)?
+        ///    The optional "alignment" component. This is a comma, followed by an optional
+        ///    minus sign, followed by one or more decimal digits.
+        ///
+        /// (\:[^\}]+)?
+        ///    The optional "format string" componet. This is a colon, followed by one or more
+        ///    characters that aren't close curly braces.
+        ///
+        /// \}
+        ///    The close curly brace indicates the end of the placeholder.
+        /// </summary>
+        private static Regex s_placeHolderRegex = new Regex(@"\{(\d+)(\,\-?\d+)?(\:[^\}]+)?\}", RegexOptions.Compiled);
+
+        /// <summary>
+        /// Returns the number of replacement strings needed to properly format the given text.
+        /// </summary>
+        public static int GetReplacementCount(string text)
+        {
+            int placeHolderCount = 0;
```

Comment:
nit: since "placeholder" is a word, it should be spelled without the capital H.

Reply:
Fixed.